lib/components/app/responsive-webapp.js

```diff
@@ -280,6 +283,14 @@ class RouterWrapperWithAuth0 extends Component {
               component={UserAccountScreen}
               path={[`${CREATE_ACCOUNT_PATH}/:step`, ACCOUNT_SETTINGS_PATH]}
             />
+            <Route
+              component={frame(components.TermsOfService)}
+              path={TERMS_OF_SERVICE_PATH}
+            />
+            <Route
+              component={frame(components.TermsOfStorage)}
```

Comment:
I'm just wondering if we really need the `AppFrame` component. Could we just keep things how they are in `AccountPage` and use

```jsx
<Route
  render={() => <AccountPage><TermsOfService /></AccountPage>}
  path={TERMS_OF_SERVICE_PATH}
/>
```

Reply:
Aren't the terms and conditions routes public?

Reply:
Oh that's true. I forgot about the HOCs that were wrapping AccountPage. In that case, yes I suppose we do need AppFrame.